css/styles.css

```diff
@@ -745,7 +745,7 @@
 }
```

Comment:
- [ ] Please make sure you give the code review yourself.🙃

- [ ] Kudos for being polite in giving the review👏👍😁. Kindly includes an explanation, a "why", for each requested change. For example, this is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*. This request lacks a "why": *Why don't you use an article tag here instead of a div?*.

- [ ] Kindly make use of inline comments to give your review. This is an example of an inline comment. In order to give inline comments, click on the files changed, and scroll to the particular line of code you want to comment on. Click on that line to add a comment. After you are done reviewing, click on the Finish your review button, add a short summary and select 'comment' or 'require changes'. Please you can check out more about the steps in [this link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request).